src/http_server/api/v1/metrics.c

```diff
@@ -181,6 +212,10 @@ void cb_metrics_prometheus(mk_request_t *request, void *data)
     long now;
     flb_sds_t sds;
     flb_sds_t sds_metric;
+    flb_sds_t tmp_sds;
+    struct flb_sds *sds_head;
```

Comment:
When I build, this is flagged as unused:

```
[ 39%] Building C object src/http_server/api/v1/CMakeFiles/api-v1.dir/metrics.c.o
/home/ec2-user/fluent-bit/src/http_server/api/v1/metrics.c: In function ‘cb_metrics_prometheus’:
/home/ec2-user/fluent-bit/src/http_server/api/v1/metrics.c:216:21: warning: variable ‘sds_head’ set but not used [-Wunused-but-set-variable]
     struct flb_sds *sds_head;
```

Reply:
relict from version long time ago, removed ;-)